lmsweb/models.py

```diff
@@ -149,9 +149,11 @@ def next_unchecked(cls):
 
     @classmethod
     def next_unchecked_of(cls, exercise_id):
-        next_exercise = cls.is_checked == False & exercise_id == cls.exercise  # NOQA: E712, E501
+        # next_exercise =   # NOQA: E712, E501
```

Comment:
? Remove

Reply:
it is written inline, putting that in other variable didn't work as expected